docusaurus/docs/core-concepts/troubleshooting/tunnel-troubleshooting.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Tunnel Troubleshooting
```

Comment:
This article is redundant to `/docusaurus/docs/core-concepts/clients/tunnelers/linux-tunnel-troubleshooting.md`.

Reply:
removed